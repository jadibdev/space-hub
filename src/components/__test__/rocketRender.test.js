import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import RocketRender from '../rocketRender';
import store from '../../redux/configureStore';

it('To check test is working', () => {
  expect(true).toBe(true);
});

it('Rocket rendering check', () => {
  const list = {
    description: 'The best', name: 'Fal', image: 'https//image1.com', reserved: true,
  };
  const tree = renderer.create(
    <Provider store={store}>
      <RocketRender List={list} />
    </Provider>
    ,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('Rocket rendering check', () => {
  const list = {
    description: 'The second besr', name: 'Et24', image: 'https//image2.com', reserved: true,
  };
  const tree = renderer.create(
    <Provider store={store}>
      <RocketRender List={list} />
    </Provider>
    ,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
