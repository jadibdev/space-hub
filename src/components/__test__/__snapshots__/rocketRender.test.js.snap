// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket rendering check 1`] = `
<div
  className="rocket"
>
  <img
    alt="Fal"
    className="img"
    src="https//image1.com"
  />
  <div
    className="rocket-dis"
  >
    <h1
      className="rocket-name"
    >
      Fal
    </h1>
    <div
      className="pharagraph"
    >
      <p>
        <span
          className="reservedText"
        >
          Reserved
        </span>
        <span
          className="description"
        >
          The best
        </span>
      </p>
    </div>
    <button
      className="btnsecond"
      onClick={[Function]}
      type="submit"
      value={true}
    >
      Cancel Reservation
    </button>
  </div>
</div>
`;

exports[`Rocket rendering check 2`] = `
<div
  className="rocket"
>
  <img
    alt="Et24"
    className="img"
    src="https//image2.com"
  />
  <div
    className="rocket-dis"
  >
    <h1
      className="rocket-name"
    >
      Et24
    </h1>
    <div
      className="pharagraph"
    >
      <p>
        <span
          className="reservedText"
        >
          Reserved
        </span>
        <span
          className="description"
        >
          The second besr
        </span>
      </p>
    </div>
    <button
      className="btnsecond"
      onClick={[Function]}
      type="submit"
      value={true}
    >
      Cancel Reservation
    </button>
  </div>
</div>
`;
